src/themes/tufte/styles.css

```diff
@@ -0,0 +1,76 @@
+@import '../../extensions/cite-apa/styles.css';
+@import '../../extensions/code/styles.css';
+@import '../../extensions/math/styles.css';
+@import '../../extensions/pages/styles.css';
+@import '../../extensions/person/styles.css';
+
+@import '../../fonts/etBook/etBook.css';
+@import 'https://unpkg.com/tufte-css@1.7.2/tufte.min.css';
```

Comment:
Something to note with this approach is that the imported CSS is not constrained to the `:--root`/`data-itemscope="root"` selector, meaning that these themes break the [first "rule" from our README](https://github.com/stencila/thema#approaches):

> All selectors must be descendants of a custom semantic selector. This reduces risks of a theme interfering with exsitng stylesheets on a website.

I'm okay with merging this as is, but would be nice to mention the caveat in the theme README at least.
We can have a conversation at a later date about how dogmatic we are about the above rule, and ways we can solve it. For example a simple one could be a PostCSS plugin to prefix the imported file selectors with `:--root`.

_This also applies to the `latex` theme._